### ISSUE TITLE
Result of NanoporeReadScanner.jar

### ISSUE THREAD

**Issue by alicecsq** [ucagenomix/sicelore#34]
Hi, 
Thank you for providing the Sicelore software. I have been using it to analyze some of my data. 

I realized I was losing a lot of reads (60.8%) at the polyA scanning step. Starting with 19,838,508 reads from a MinION flow cell, I ended up with 7771672 reads (39.2%) having polyA/T and adaptor found after the polyA scanning. This significant loss of reads is surprising to me as the paper reported 57 ± 11% of the reads to be having polyA/T and adaptor. I am wondering 1) if you may have some ideas why this is happening; 2) if you are planning to release the source code, especially the NanoporeReadScanner-0.5.jar; if so, when would that rough date be?

I did notice that in #15, you suggested doing the optional biotin selection step. I am still interested to know, but haven't figured out, what the potential 10xGenomics PCR byproducts (you referred to) are. Could you help clarify? (Or I very much appreciate it if you could point me to a paper/reference. 

My final question is about the published protocol, saying that 'the Ns at the 5′ end of the PCR primers avoid biasing towards reverse Nanopore reads generation'.  I am curious to know what this idea is coming from, or how does adding this Ns help?

I am sorry for the laundry list of questions, and specially appreciate your patience reading this. Thank you so much!

Best,
Alice

**Comment by RainerWaldmann** [comment]
Hi Alice,

The PCR in the 10x Genomics single cell workflow generates to a variable extend PCR artefacts that don't have a polyA (e.g. PCR products with a TSO at either end).  We never tried to figure out how they are generated since to avoid them we would need to modify the 10x Genomics protocol. We know that those are mainly PCR products with an TSO at either end (at least for the samples for which we examinated this). This is variable and can reach 60% and even more .   We chose to remove most of those PCR artefacts during the Nanopore library preparation.  I would highly recommend that you do the PCR with the  biotinylated primers  and the Streptavidin enrichment which removes most of the PCR products that don't have polyA tails.

Regarding the N's at the end of primers. DNA ligases have bias, they ligate some ends better than others. When we realized that we obtained mainly reads in one direction we suspected a ligation bias. The N's at the end of primers generate heterogeneity at the ends of the PCR products and remove this bias. 

Best,

Rainer
